internal/controller/replication.storage/status.go

```diff
@@ -26,18 +26,21 @@ import (
 // sets conditions when volume was promoted successfully.
 func setPromotedCondition(conditions *[]metav1.Condition, observedGeneration int64) {
 	setStatusCondition(conditions, &metav1.Condition{
+		Message:            "Volume is Promoted",
 		Type:               v1alpha1.ConditionCompleted,
 		Reason:             v1alpha1.Promoted,
 		ObservedGeneration: observedGeneration,
 		Status:             metav1.ConditionTrue,
 	})
 	setStatusCondition(conditions, &metav1.Condition{
+		Message:            "Volume is Healthy",
 		Type:               v1alpha1.ConditionDegraded,
 		Reason:             v1alpha1.Healthy,
 		ObservedGeneration: observedGeneration,
 		Status:             metav1.ConditionFalse,
 	})
 	setStatusCondition(conditions, &metav1.Condition{
+		Message:            "Volume is not resyncing",
```

Comment:
the action/state in the message field are inconsistent in terms of capitalization. It would be better to follow the same everywhere? either capitalize or leave them in lowercase. WDYT?
```suggestion
		Message:            "Volume is not Resyncing",
```

Reply:
all of them seems to be in lower case already?
Am I missing it somewhere?

Reply:
ohh I see, couple of states were capital, modified them.